src/sky/pro/java/course2/StringListImpl.java

```diff
@@ -0,0 +1,180 @@
+package sky.pro.java.course2;
+
+import sky.pro.java.course2.exceptions.AddingNullException;
+import sky.pro.java.course2.exceptions.IndexOutBoundsException;
+import sky.pro.java.course2.exceptions.NotFound;
+import sky.pro.java.course2.exceptions.NullPointerException;
+
+public class StringListImpl implements StringList {
+    String[] arrayString;
+    int indexOfElement = 0;
```

Comment:
не забываем про модификаторы доступа
```suggestion
    private String[] arrayString;
    private int indexOfElement = 0;
```